scheduler/setup_funcs.go

```diff
@@ -27,8 +28,29 @@ func cacheTaskGroups(comparator *CmpBasedTaskComparator) error {
 	comparator.projects = make(map[string]project)
 	for _, v := range comparator.versions {
 		p := project{}
-		if err := yaml.Unmarshal([]byte(v.Config), &p); err != nil {
-			return errors.Wrapf(err, "error unmarshalling task groups from version %s", v.Id)
+		if v.ParserProject != nil {
```

Comment:
Since what you need ultimately is a subset of `model.Project`, could you call `LoadProjectFromVersion` to populate a `model.Project` and delete `scheduler.project` to simplify this code?

Reply:
Well it seems as though he has scheduler.project to speed up the code. Would there be performance issues involved with reading the entire model.Project?

Reply:
I think you can ignore my comment here. Sam is working on this in a separate ticket.